src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig

```diff
@@ -213,8 +217,12 @@
                             </p>
                             <ul>
                                 {% for key, value in requirements.testsOptional %}
-                                    {% if 'fail' == value and testsErrors[key] is defined %}
-                                        <li>{{ requirements.testsErrors[key] }}</li>
+                                    {% if 'fail' == value %}
+                                        {% if testsErrors[key] is defined %}
+                                            <li>{{ requirements.testsErrors[key] }}</li>
+                                        {% else %}
+                                            <li>{{ '%key% (missing description)'|trans({'%key%': key}, 'Admin.Advparameters.Notification') }}</li>
```

Comment:
Must be in _Admin.Advparameters.Feature_.

Reply:
Wy? it's a notification :(

Reply:
Because we classified the basic content of a page in the Feature file.